src/core/entities/MRModelEntity.js

```diff
@@ -124,7 +136,9 @@ export class MRModelEntity extends MRDivEntity {
 
             this.loading = false;
 
-            if (modelChange) {
+            this.loaded = true;
```

Comment:
having both of these as variables can be a little confusing, can you change one of their names to be more distinct (or if one of them is no longer used, remove the redundant one)

Reply:
forgot to highlight which variables - 'this.loading' vs 'this.loaded'

Reply:
@hanbollar I agree, I sat on it for a minute and I think I might replace it with a status enum with three potential values (not loaded, loading, loaded) when I dive into the instancing stuff, as it will likely fundamentally change howe we initialize models.

Reply:
that makes sense

Reply:
just adding a todo comment for here then is fine for this pr

Reply:
I added a TODO comment. I plan on tackling instancing next week.